yarn-project/builder/src/contract-interface-gen/codegen.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
```

Comment:
You can sometimes use `git mv` to rename files and make it easier for git to understand that they were just moved and not changed - the diff currently lists the old file as removed and this being a brand new file, so it's hard to review if anything changed.

Reply:
This particular file was heavily modified after moving, because it used the "sync" version of the methods for everything so I refactored it to be non-blocking